client/src/main/java/io/pravega/client/state/RevisionedStreamClient.java

```diff
@@ -101,4 +101,16 @@
     @Override
     abstract void close();
 
+    /**
+     * Read all data from given start revision  to the end of the stream. The iterator returned will
```

Comment:
Java doc for this method should be changed. Read all data between given start revision and end revision(not end of the stream). Iterator will stop once it reaches to end revision.

Reply:
ok